### PR TITLE
Modules update

### PR DESCRIPTION
**Pull request by Alonof**
update SD card for:
MTB_MXCHIP_EMW3166
MTB_USI_WM_BN_BM_22
MTB_ADV_WISE_1530

### CODE REVIEW

File: config/mbed_lib.json [hunk 1]
```diff
@@ -167,6 +167,24 @@
...
+            "SPI_MOSI": "PB_15",
+            "SPI_MISO": "PB_14",
+            "SPI_CLK":  "PB_13",
+            "SPI_CS":   "PA_12"
+        },
+        "MTB_USI_WM_BN_BM_22": {
+            "SPI_MOSI": "PC_3",
+            "SPI_MISO": "PC_2",
+            "SPI_CLK":  "PB_13",
+            "SPI_CS":   "PC_5"
```
Comment (geky) [inline config/mbed_lib.json]: Oh, does MTB_USI_WM_BN_BM_22 / MTB_MXCHIP_EMW3166 have an SD card slot?

Comment (geky) [reply config/mbed_lib.json]: Asked offline, answer is yes :+1:

File: config/mbed_lib.json [hunk 2]
```diff
@@ -167,6 +167,24 @@
             "SPI_MISO": "p6",
             "SPI_CLK": "p7",
             "SPI_CS": "p8"
+        },
+        "MTB_ADV_WISE_1530": {
+            "SPI_MOSI": "NC",
```
Comment (deepikabhavnani) [inline config/mbed_lib.json]: Target override is for devices that have SD interface. Any reason to have `NC` here?

Comment (geky) [reply config/mbed_lib.json]: Ah, this is a workaround for a bug @bridadan is fixing.

At the moment any libraries with multiple storage options (ie storage-selector) will fail to compile if D1/D0 is not defined.

This overload will work as a temporary solution, @bridadan has plans to change the default pins to NC. (@bridadan when did you plan to do that?)

Comment (bridadan) [reply config/mbed_lib.json]: ...soooon 😄 I hadn't started on that yet, a few things came up. I guess we just need to submit PRs against all of the storage options (we can just use the list in storage-selector).

Comment (deepikabhavnani) [reply config/mbed_lib.json]: @bridadan - Is it fine to have this fix temporary till you add a proper solution?

Comment (bridadan) [reply config/mbed_lib.json]: absolutely!

**Comment by deepikabhavnani** [comment]
@Alonof - Is basic read/write to SD card tested with some application?

**Comment by Alonof** [comment]
Not yet
I will test the SD card very soon

**Pull request closed without merging**